### ISSUE TITLE
ChannelManager#putPermissionOverride drops NPE for no reason

### ISSUE THREAD

**Issue by VitorBlog** [DV8FromTheWorld/JDA#1375]
[download]: https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion
[guild]: https://discord.gg/0hMr4ce0tIk3pSjp
[stack overflow]: https://stackoverflow.com/questions/tagged/java

## General Troubleshooting

  
- [ ] I have checked for similar issues.
- [X] I have updated to the [latest JDA version][download].
- [ ] I have checked the branches or the maintainers' PRs for upcoming bug fixes.

## Bug Report

When you try to add a permissionOverride drops a NPE

### Expected Behavior

Add a member to a channel

### Code Example or Reproduction Steps

```
textChannel.manager.putPermissionOverride(
                member,
                3526208L,
                135168L
            ).queue() // <- Exception line

// Same result using list of permissions
```

that code work's

```
textChannel
                .createPermissionOverride(member)
                .setAllow(3526208L)
                .setDeny(135168L)
                .queue()
```

### Code for JDABuilder or DefaultShardManagerBuilder Used

```
JDABuilder.createDefault(config.getString("bot.token"))
            .addEventListeners(DiscordHandler())
            .build()
            .awaitReady()
```

### Exception or Error

```
[ForkJoinPool.commonPool-worker-2] ERROR RestAction - Encountered error while processing success consumer
java.lang.NullPointerException
    at net.dv8tion.jda.internal.requests.restaction.PermOverrideData.<init>(PermOverrideData.java:46)
    at net.dv8tion.jda.internal.managers.ChannelManagerImpl.lambda$getOverrides$10(ChannelManagerImpl.java:401)
    at gnu.trove.map.hash.TLongObjectHashMap.forEachEntry(TLongObjectHashMap.java:417)
    at gnu.trove.impl.sync.TSynchronizedLongObjectMap.forEachEntry(TSynchronizedLongObjectMap.java:153)
    at net.dv8tion.jda.internal.managers.ChannelManagerImpl.getOverrides(ChannelManagerImpl.java:396)
    at net.dv8tion.jda.internal.managers.ChannelManagerImpl.lambda$finalizeData$9(ChannelManagerImpl.java:369)
    at net.dv8tion.jda.internal.managers.ManagerBase.withLock(ManagerBase.java:156)
    at net.dv8tion.jda.internal.managers.ChannelManagerImpl.finalizeData(ChannelManagerImpl.java:366)
    at net.dv8tion.jda.internal.requests.RestActionImpl.queue(RestActionImpl.java:182)
    at net.dv8tion.jda.internal.managers.ManagerBase.queue(ManagerBase.java:121)
    at net.dv8tion.jda.api.requests.RestAction.queue(RestAction.java:411)
    at net.dv8tion.jda.api.requests.RestAction.queue(RestAction.java:377)
    at com.vitorblog.shopbot.command.ProposalCommand$acceptProposal$1.accept(ProposalCommand.kt:227)
    at com.vitorblog.shopbot.command.ProposalCommand$acceptProposal$1.accept(ProposalCommand.kt:24)
    at net.dv8tion.jda.api.requests.Request.lambda$onSuccess$0(Request.java:95)
    at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
    at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
    at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
    at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
    at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
```

**Comment by MCOfficer** [comment]
you can probably work around this by caching members.

The error occurs here:
https://github.com/DV8FromTheWorld/JDA/blob/3a9fcd0ecc88f988ac81eb2c18527283a83f12d1/src/main/java/net/dv8tion/jda/internal/requests/restaction/PermOverrideData.java#L46

And getMember is nullable if the Member isn't cached:
https://github.com/DV8FromTheWorld/JDA/blob/3a9fcd0ecc88f988ac81eb2c18527283a83f12d1/src/main/java/net/dv8tion/jda/api/entities/PermissionOverride.java#L121

**Comment by MinnDevelopment** [comment]
Fixed in **4.2.0_194**
